### DISCUSSION TITLE
Mojolicious and Vite

### DISCUSSION THREAD

**Discussion by tarkhil** [mojolicious/mojo#2020] [General]
Hello

I'm working on a project with complex Vue-based interface, and I see that Vite and Mojolicious has quite different views on directory structure. 

MAYBE I'll just split a project into independent front and back; I'm still not sure. However, my current view is something like "Mojolicious handles routes like / /login /logout /api/* and all Vite-generated are just static". 

Can anyone share their experience, if any?

**Comment by rabbiveesh** [comment]
I do this, and I have a separate frontend directory for all of the vue code.

I then build it and serve it out of the public directory as static assets.

**Reply by yusrideb** [reply]
I use more or less the same method as this in ReactJS.
